src/redux-store/slices/authSlice.js

```diff
@@ -23,29 +24,26 @@ export const authSlice = createSlice({
     builder
       .addCase(register.pending, handlePending)
       .addCase(register.rejected, handleRejected)
-      .addCase(register.fulfilled, (state, action) => ({
-        ...state,
-        token: action.payload.token,
-        isLoggedIn: true,
-      }))
+      .addCase(register.fulfilled, (state, action) => {
```

Comment:
Is it doesn't work without this changes? I just try to understand if we don't do this changes here it is not working like we want it?